Sources/BraintreeDropIn/BTPaymentSelectionViewController.m

```diff
@@ -396,17 +394,14 @@ - (UICollectionViewCell *)collectionView:(UICollectionView *)savedPaymentMethods
     BTPaymentMethodNonce *paymentInfo = self.paymentMethodNonces[indexPath.row];
     cell.paymentMethodNonce = paymentInfo;
     NSString *typeString = paymentInfo.type;
-    NSMutableAttributedString *typeWithDescription = [[NSMutableAttributedString alloc] initWithString:[NSString stringWithFormat:@"%@", paymentInfo.localizedDescription ?: @""]];
-    if ([paymentInfo isKindOfClass:[BTCardNonce class]]) {
-        typeWithDescription = [[NSMutableAttributedString alloc] initWithString:[NSString stringWithFormat:@"••• ••%@", ((BTCardNonce*)paymentInfo).lastTwo ?: @""]];
-    }
+    
     cell.highlighted = NO;
-    cell.descriptionLabel.attributedText = typeWithDescription;
```

Comment:
As far as we could tell, there were no attributes being added to this text, so we changed it from `attributedText` to `text`.